lib/registration.pm

```diff
@@ -566,6 +566,10 @@ sub process_scc_register_addons {
 }
 
 sub process_scc_register {
+    # prevent rogue RMT servers to show up in unexpected selection dialogs
+    # https://progress.opensuse.org/issues/94696
+    set_var('REG_URL', 'https://scc.suse.com') unless get_var('REG_URL');
```

Comment:
it happens before the registration process.

Reply:
Correct. Thank you. Updated just now to set the variable before typing the setting into the bootloader command line